manifests/m/Microsoft/dotnetRuntime/5-x64/5.0.9.30315/Microsoft.dotnetRuntime.5-x64.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetRuntime.5-x64
+PackageVersion: 5.0.9.30315
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/8bc41df1-cbb4-4da6-944f-6652378e9196/1014aacedc80bbcc030dabb168d2532f/windowsdesktop-runtime-5.0.9-win-x64.exe
+  InstallerSha256: EA3EC92C5DD0C6A396977B1FDE0CEB510B3D82BEC37ACF24C6BF8ACD705D9647
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive

```

Same InstallerType on this one 😁
```suggestion
  InstallerType: burn
  Scope: machine
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/8bc41df1-cbb4-4da6-944f-6652378e9196/1014aacedc80bbcc030dabb168d2532f/windowsdesktop-runtime-5.0.9-win-x64.exe
  InstallerSha256: EA3EC92C5DD0C6A396977B1FDE0CEB510B3D82BEC37ACF24C6BF8ACD705D9647
```